symplyphysics/laws/electricity/circuits/resistance_in_serial_connection.py

```diff
@@ -0,0 +1,55 @@
+"""
+Resistivity of serial resistors
+===============================
+
+The total resistance of the circuit whose components are connected in series is the sum
+of the resistances of individual components.
+
+**Conditions:**
+
+#. The circuit is a DC one or the reactance is zero.
```

Comment:
Maybe decipher `DC`? Also it is characterstic of power source, not circuit.

Reply:
No, it is a standard definition:

> A **direct current circuit** is an electrical circuit that consists of any combination of constant voltage sources, constant current sources, and resistors.

Reply:
But I'll remove the abbreviation.